core/modfunc.go

```diff
@@ -101,6 +100,15 @@ func (fn *ModuleFunction) Call(ctx context.Context, caller *idproto.ID, opts *Ca
 	}
 	ctx = bklog.WithLogger(ctx, lg)
 
+	props := map[string]any{
+		"function_name": fn.metadata.Name,
+	}
+	moduleAnalyticsProps(mod, "", props)
+	if caller, err := mod.Query.CurrentModule(ctx); err == nil {
+		moduleAnalyticsProps(caller, "caller_", props)
+	}
+	analytics.Ctx(ctx).Capture(ctx, "module_call", props)
+
```

Comment:
@vito @sipsma Can you double check this here?

Also -- thoughts on how to best prefix/name caller vs callee?